Detectors/GlobalTrackingWorkflow/tofworkflow/src/RecoWorkflowSpec.cxx

```diff
@@ -139,9 +135,6 @@ class TOFDPLRecoWorkflowTask
     pc.outputs().snapshot(Output{o2::header::gDataOriginTOF, "CALIBINFOS", 0, Lifetime::Timeframe}, mMatcher.getCalibVector());
 
     // declare done
-    finished = true;
-    //pc.services().get<ControlService>().readyToQuit(QuitRequest::Me);
-    pc.services().get<ControlService>().endOfStream();
```

Comment:
so how is this processor shutting itself down?

Reply:
It should stop when the previous processors (the ones providing input for it) send their endOfStream. Am I wrong? Anyway, it seems to work in this way.